jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -166,6 +174,7 @@
            - raw
            - rbd
            - netapp
+           - nfs
```

Comment:
@jdsn Yes. So do you want an additional check whether the steps are part of ```$mkcloudtarget```, I can do that. But I think trying to dynamically insert those steps in the correct place would be a bad idea.